packages/@okta/vuepress-site/docs/guides/oin-api-service-overview/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: API service integrations in the OIN
+meta:
+  - name: description
+    content: Provides an overview of why and when to build API service integrations for the Okta Integration Network.
+---
+
+<ApiLifecycle access="ea" />
+
+> **Note:** The API Service Integration is a [Self-service Early Access (EA)](/docs/concepts/feature-lifecycle-management/#self-service-features) feature. Enable this feature in your org to obtain the API service integration capability.
+<!-- Need OAUTH_ADMIN_CONSENT_DIALOG FF enabled in monolith org -->
+
+Service apps that access or modify Okta resources, like system logs, apps, sessions, and policies can be published in the Okta Integration Network (OIN) as an API service integration. This integration type allows your service app to access your customer tenant Okta org through [Okta management APIs](/docs/reference/core-okta-api/) using OAuth 2.0.
+
+Previously, if you have a service app requiring Okta management API access, you need to use an API token that was tied to a specific Okta user and you can't restrict access to certain resources. With an OAuth 2.0 API service integration, your service app can have secure, reliable, and granular access to Okta APIs without being associated with a user.
+
+When your service app integration is in the Okta Integration Network (OIN), your customers can discover and configure your integration through the OIN catalog in their own Okta tenant org. Configuration is easy and consistent for your customers because you've already done the hard work by building the integration with the required configuration instructions. Customers trust that integrations in the OIN are secure and reliable because it's Okta verified.
```

Comment:
Thanks for the suggestion! I have to inject the "integration" word in the first sentence because the app is not in the catalog, it's the integration.
```suggestion
When your service app integration is listed in the Okta Integration Network (OIN) catalog, your customers can discover and configure it in their own Okta tenant org. Configuration is easy and consistent for your customers because you've already done the hard work by building the integration with the required configuration instructions. Customers trust that integrations in the OIN are secure and reliable because they're Okta-verified.
```